source/lex.tex

```diff
@@ -1327,14 +1327,17 @@
 cannot be represented by any type in its list and
 an extended integer type\iref{basic.fundamental} can represent its value,
 it may have that extended integer type.
-If all of the types in the list for the \grammarterm{integer-literal}
+This type shall have the following signedness:
+\begin{itemize}
+\item If all types in the list for the \grammarterm{integer-literal}
 are signed,
 the extended integer type is signed.
-If all of the types in the list for the \grammarterm{integer-literal}
+\item If all types in the list for the \grammarterm{integer-literal}
 are unsigned,
 the extended integer type is unsigned.
-If the list contains both signed and unsigned types,
+\item Otherwise,
```

Comment:
This "otherwise" feels like it might refer to the immediately preceding "if" only.
When we do if/otherwise ladders, we put "otherwise" on each subsequent "if".

In this case, I'd just leave the text of the third option as-is.

Reply:
Done. I've somewhat simplified the bullets though, since repeating the term "extended integer type" in each is needlessly verbose.